log/log.go

```diff
@@ -79,3 +82,26 @@ func init() {
 	log.Out = os.Stdout
 	log.SetLevel(logrus.DebugLevel)
 }
+
+// GetLogger get logrus.Logger object accoring to logName
+// each module can have it's own logger
+func GetLogger(logName string, bConsole bool) *logrus.Logger {
+	curLog, ok := logMap[logName]
```

Comment:
we can't return *logrus.Logger to external, because we may change log implementation later

Reply:
So, need create a log wrapper.
Delete all logs at this time